### ISSUE TITLE
Not compatible with new SslStream implementation

### ISSUE THREAD

**Issue by yannleretaille** [back2mach/apns#2]
It seems that the devs from rust-openssl [completely rewrote the SslStream implementation](https://github.com/sfackler/rust-openssl/commit/91f8c542f75ff81ce7f3f12f26c469127ff0ab18#diff-e6acc97c6222cbc160517e2286040f19R903) two weeks ago and now apns does not compile anymore with the default dependecies.

```
   Compiling openssl v0.7.4
   Compiling openssl-sys-extras v0.7.4
   ....
   Compiling apns v0.1.0
~/.multirust/toolchains/nightly/cargo/registry/src/github.com-88ac128001ac3a9a/apns-0.1.0/src/apns/mod.rs:412:9: 412:23 error: no associated item named `new` found for type `openssl::ssl::SslStream<_>` in the current scope
~/.multirust/toolchains/nightly/cargo/registry/src/github.com-88ac128001ac3a9a/apns-0.1.0/src/apns/mod.rs:412   return SslStream::new(&context, tcp_conn);
                                                                                                                               ^~~~~~~~~~~~~~
error: aborting due to previous error
```

**Comment by back2mach** [comment]
Thanks! I haven't updated my local crates for a while.
The issue should been fixed in version 0.1.1. Feel free to checkout!

**Comment by yannleretaille** [comment]
that was quick, thanks!
